pkg/controller/dgraph/dgraph.go

```diff
@@ -135,13 +139,27 @@ func ExecuteQuery(query string, root interface{}) error {
 func MutateNode(n []byte) (*api.Assigned, error) {
 	mu := &api.Mutation{
 		CommitNow: true,
+		SetJson:   n,
 	}
 
-	mu.SetJson = n
 	ctx := context.Background()
 	return client.NewTxn().Mutate(ctx, mu)
 }
 
+// DeleteNodes batch deletes given nodes
```

Comment:
Yes. Just one difference for creating/updating a nod(MutateNode func) and deleteting nodes(DeleteNodes func) instead of using `SetJson` field delete func will use `DeleteJson` field in `api.Mutation` struct

Reply:
Is it better to have just just one function with another argument which determines the type of mutation i.e, create/update or delete.